### PR TITLE
Fix dob date range valdiation and picker restriction

### LINKED ISSUE #83: DOB picker error shows timestamp instead of date
image

## Reproduce

Pick a DOB later than today

## Expect

Error shows more readable date

### PR DESCRIPTION
**Pull request by vitokhangnguyen**
Closes #83

### CODE REVIEW

File: src/models/constnats.ts [hunk 1]
```diff
@@ -1,3 +1,6 @@
+export const Default_Date_Format = 'DD-MMM-YYYY';
```
Comment (Andrewnt219) [inline src/models/constnats.ts:1]: ALL_CAPS for constants.

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

File: src/models/constnats.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
-export const Genders = {
+export const DEFAULT_DATE_FORMAT = 'DD-MMM-YYYY';
```
Comment (Andrewnt219) [inline src/models/constnats.ts:1]: Should we do this like the genders?

```ts
export enum Format {
  SHORT_DATE = 'MM-DD',
  SHORT_TEXT_DATE = 'MMM DD',
  DATE = 'YYYY-MM-DD',
  DATE_TIME = 'YYYY-MM-DD h:mm a',
  DATE_TIME_TEXT = 'MMMM DD, YYYY - h:mm a',
  DATE_TEXT = 'MMMM DD, YYYY',
  TIME = 'h:mm a',
}

dayjs().format(Format.DATE_TIME_TEXT)
```

File: src/models/constnats.ts [hunk 3]
```diff
@@ -1,6 +1,25 @@
...
+  MEDIUM_DATE: 'DD-MM-YYYY', // 31-12-2021
+  MEDIUM_DATE_TEXT: 'MMM DD, YYYY', // Dec 31, 2021
+
+  LONG_DATE: 'DD-MMM-YYYY', // 31-Dec-2021
+  LONG_DATE_TEXT: 'MMMM DD, YYYY', // December 31, 2021
+
+  EXTRA_LONG_DATE: 'ddd, DD-MM-YYYY', // Fri, 31-12-2021
+  EXTRA_LONG_DATE_TEXT: 'dddd, MMMM DD, YYYY', // Friday, December 31, 2021
+
+  DATE_TIME_12H: 'YYYY-MM-DD h:mm a', // 2021-12-31 11:59 PM
```
Comment (Andrewnt219) [inline src/models/constnats.ts:16]: This one is different from the other day-month-year.

**Review by Andrewnt219 — CHANGES_REQUESTED** [review]

File: src/models/constnats.ts (hunk 3 above)
Comment (vitokhangnguyen) [reply src/models/constnats.ts:16]: Oops, for got this. Fixed!

**Comment by vitokhangnguyen** [comment]
@Andrewnt219  Let's use the `LONG_DATE` format for the date picker

**Review by Andrewnt219 — APPROVED** [review]

**Pull request merged**